### PR TITLE
Add Vietnamese translation from Settings toggle

### PR DESCRIPTION
**Pull request by CongLeSolutionX**
Add Vietnamese translation from Settings toggle.

Below is the demo of the translated UI screens:

|                     |                     |
|---------------------|---------------------|
| Demo_Settings-Vietnamese-Translation-1 | Demo_Settings-Vietnamese-Translation-2 |

Below is the video demo with the translated UI elements:

Demo_Settings-Vietnamese-Translation-ezgif com-video-to-gif-converter

## Summary by CodeRabbit

- **New Features**
	- Added Vietnamese (Tiếng Việt) as a new language option in the application.
	- Introduced a new localization file for Vietnamese, enhancing accessibility for Vietnamese-speaking users.

- **Bug Fixes**
	- No specific bug fixes were included in this release.

- **Documentation**
	- Updated translations to include Vietnamese in the system.

### CODE REVIEW

File: renderer/locales/vi.json [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+        },
+        "digital-art-4x": {
+          "NAME": "Nghệ thuật Kỹ thuật số",
+          "DESCRIPTION": "Dành cho nghệ thuật kỹ thuật số và minh họa."
+        }
+      }
+    },
+    "DOUBLE_UPSCAYL": {
+      "TITLE": "Double Upscayl",
+      "DESCRIPTION": "Kích hoạt tùy chọn này để chạy upsayl hai lần trên một hình ảnh. Lưu ý rằng điều này có thể làm tăng đáng kể thời gian xử lý và có thể gây ra vấn đề hiệu suất cho các tỷ lệ lớn hơn 4X."
```
Comment (NayamAmarshe) [inline renderer/locales/vi.json:149]: Typo in this line.

File: renderer/locales/vi.json [hunk 2]
```diff
@@ -0,0 +1,286 @@
...
+      "TITLE": "Lỗi Đọc/Ghi",
+      "DESCRIPTION": "Đảm bảo rằng đường dẫn là chính xác và bạn có quyền đọc/ghi thích hợp \n({data})"
+    },
+    "TILE_SIZE_ERROR": {
+      "TITLE": "Lỗi",
+      "DESCRIPTION": "Kích thước gạch không đúng. Vui lòng thay đổi kích thước gạch trong cài đặt hoặc đặt là 0 ({data})"
+    },
+    "EXCEPTION_ERROR": {
+      "TITLE": "Lỗi Ngoại lệ",
+      "DESCRIPTION": "Upscayl gặp lỗi. Có thể, nhị phân upsayl không thực hiện các lệnh đúng cách. Hãy kiểm tra nhật ký để xem bạn có thông tin gì không. Bạn có thể đăng vấn đề trên kho lưu trữ GitHub của Upscayl để được giúp đỡ thêm."
```
Comment (NayamAmarshe) [inline renderer/locales/vi.json:232]: Typo here

**Comment by NayamAmarshe** [comment]
Thanks for the PR! Can we have a Vietnamese person confirm these translations?

**Comment by CongLeSolutionX** [comment]
> Thanks for the PR! Can we have a Vietnamese person confirm these translations?

Thanks for your response!
I'm Vietnamese and bilingual.
Let me update the typos.

**Comment by NayamAmarshe** [comment]
Thank you for the changes :D

**Pull request merged**